src/components/core/Headline.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import noop = require('lodash/noop');
+import { HeadlineTransformation } from '@gooddata/indigo-visualizations';
+import { Execution } from '@gooddata/typings';
+
+import { IntlWrapper } from './base/IntlWrapper';
+import { IntlTranslationsProvider, ITranslationsComponentProps } from './base/TranslationsProvider';
+import { fixEmptyHeaderItems } from './base/utils/fixEmptyHeaderItems';
+import { HeadlinePropTypes, Requireable } from '../../proptypes/Headline';
+
+import { ErrorStates } from '../../constants/errorStates';
+
+import { IBaseVisualizationProps, IBaseVisualizationState, BaseVisualization } from './base/BaseVisualization';
+
+export { Requireable };
+
+const defaultErrorHandler = (error: any) => {
+    if (error && error.status !== ErrorStates.OK) {
+        console.error(error); // tslint:disable-line:no-console
+    }
+};
+
+export class Headline extends BaseVisualization<IBaseVisualizationProps, IBaseVisualizationState> {
+    public static defaultProps: Partial<IBaseVisualizationProps> = {
+        resultSpec: {},
+        onError: defaultErrorHandler,
+        onLoadingChanged: noop,
+        ErrorComponent: null,
+        LoadingComponent: null,
+        afterRender: noop,
+        pushData: noop,
+        locale: 'en-US',
```

Comment:
Is this locale specified in every component separately? This could be in some imported file.